frontend/components/base/BasePagination.vue

```diff
@@ -195,13 +195,10 @@ export default {
     },
     keyDown(event) {
       if (!this.paginationSettings.disabledShortCutPagination) {
-        const elem = document.querySelector("body");
-        if (elem === document.activeElement) {
-          if (event.keyCode === 39) {
-            this.nextPage();
-          } else if (event.keyCode === 37) {
-            this.prevPage();
-          }
+        if (event.keyCode === 39) {
+          this.nextPage();
+        } else if (event.keyCode === 37) {
+          this.prevPage();
```

Comment:
you can be more explicit by putting constants for the keycode for arrowRight and arrowLeft. For later to know what means these code